### ISSUE TITLE
Bone makes skeletal origin change on different angle incorrectly

### ISSUE THREAD

**Issue by merlin2v** [synfig/synfig#632]
![glitch](https://user-images.githubusercontent.com/25653109/46051844-3e9d9980-c0f0-11e8-99dd-166398686bc3.gif)
**Synfig version & platform**: Both Dev and stable
(we're more likely to help if you're using latest development builds)
![image](https://user-images.githubusercontent.com/25653109/46051601-1c574c00-c0ef-11e8-93e3-f6410c156d3d.png)

**Issue description**:
bones are acting weird and seem to be calculating something(possibly the origin) wrong. 
the bone acts as if the origin changes depending on its angle
~~Any Ideas on how to get around this, if it is a bug? or if I am just doing something wrong?~~

I was able to find the problem which was that it seems the skeleton is not including the offset due by transformations. the fix is to make sure the origin and transformation vectors(of the folders) are set to zero, select  all the points and move them back to were they are suppose to be. this is a temporary workaround the problem. if precision is required then the mathematical vector values can be transformed manually by adding the transition vector(the value before setting it to zero) to each vector value that you have.

**Comment by Keyikedalube** [comment]
Since you didn't provide the file I can't exactly make out what caused that behavior.

However, this is easily reproducible (because I've encountered this too) if the user tweaks the group layer transformation to -ve values.
Maybe you duplicated the right arm to produce the left arm, applied -ve value to x scale property to mirror the right arm, and then linked the left arm to the bone.

Below is a sample bone file I created to replicate your issue:
[bone-group.sif.zip](https://github.com/synfig/synfig/files/6954346/bone-group.sif.zip)


1. Created right bone
2. Duplicated right bone
3. Set duplicated group scale value `x` to `-x`
4. Link bone to step 4 region layer
5. Play with linked left bone
6. Weird left bone transformation
